### ISSUE TITLE
UX: Embed actual tweets if we don't have a YouTube video but do have a twitter URL?

### ISSUE THREAD

**Issue by ramezn** [jasonLaster/abuses#33]
For incidents where we don't have a YouTube video (because there's no video, or because Jason hasn't put it in YouTube yet, or because we're copyright blocked, etc..  should the UX support directly embedding the tweet?  @pablodefendini  - I think this is for you to think about.  

While we're thinking about this, should we think about other kinds of embeddings?

**Comment by pablodefendini** [comment]
thinking out loud: a twitter embed probably wouldn't be too visually heavy, especially now that we've moved each incident onto its own page (in @michielkoning's latest pass).

**Comment by michielkoning** [comment]
Thats true, but i think it will create duplicate content. The videos and text are already visible on the detailpage. I can add it in a prototype, so we can review the changes.

**Comment by pablodefendini** [comment]
This is for incidents where we don't have a video, for whatever reason. Otherwise I think you're right that it's unnecessary for the default view — we have the link to the source tweet in there already. 

No need to prototype it yet, let's continue discussing and them prioritize it adequately.

**Comment by michielkoning** [comment]
Ahh, missed that part. Makes sense :)
